### ISSUE TITLE
Eqn 2.18: Formatting is messed up

### ISSUE THREAD

**Issue by wei2912** [learnyou/lysa#9]
I am not sure if this is intended, but the positioning of the expressions is irregular.

**Comment by pharpend** [comment]
On Tue, Jul 07, 2015 at 08:31:05AM -0700, Ng Wei En wrote:

> I am not sure if this is intended, but the positioning of the
> expressions is irregular.

I'm not entirely certain what you mean here. I noticed that the equation
number is position irregularly. Beyond that, I can't see any
irregularity.

In this case, the equation is very wide. Thus, if the equation number
were to be in its usual position, it would either be overwriting the
equation, or in the margin. The next best place is right below the
equation.

There might be a larger issue here: the equation is too huge.

Peter Harpending

**Comment by wei2912** [comment]
Equation 2.18 looks like this:

```
{..........................} =
         {.................} = {..........................}
                             = {.................}
```

which looks pretty weird to me. I'd have expected to see an equation formatted this way:

```
  {..........................}
= {.................}
= {..........................}
= {.................}
```

**Comment by pharpend** [comment]
I do think that's better.

**Comment by pharpend** [comment]
Fixed in 454556b
